Resources/Prototypes/_LostParadise/Recipes/Lathes/security.yml

```
@@ -143,3 +143,44 @@
+  completetime: 1
+  materials:
+    Steel: 1200
+
+- type: latheRecipe
+  id: LPPWeaponGoliathSniper
+  result: LPPWeaponGoliath
+  completetime: 15
+  materials:
+    Plastic: 2000
+    Steel: 10000
+    Glass: 2500
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)